logic/src/commonMain/kotlin/com/wire/kalium/logic/data/featureConfig/FeatureConfigEventEnums.kt

```diff
@@ -0,0 +1,13 @@
+package com.wire.kalium.logic.data.featureConfig
+
+// we didn't follow name convention because Enum.valueOf() only checks the constant name
+@Suppress("EnumNaming")
+enum class FeatureConfigStatus {
+    enabled,
+    disabled
+}
+
+@Suppress("EnumNaming")
+enum class FeatureConfigName {
+    fileSharing
+}
```

Comment:
Same here.
Network should be responsible for mapping it to a strongly-typed enum. Not `:logic`:

```kotlin
@Serializable
enum class FeatureConfigNameDTO {
    @SerialName("fileSharing") FILE_SHARING
}
```